app/views/entities/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<%= form_with model: @entity, url: entities_path, class: "form" do |f| %>
+   
+  <nav class="navbar navbar-default navbar-expand-lg fixed-top">
+    <div class="container flex justify-content0-between align-items-center">
+
+      <div></div>
+     
+      <h6 class="title">ADD TRANSACTION</h6>
+
+       <%= f.submit "Submit" %>
+
+    </div>
+  </nav>
+
+      <div class="d-flex justify-content-start align-items-center bg-white mav-header">
+
+        <%= link_to :back do  %> 
+          <i class="fa-solid fa-chevron-left icon" ></i>
+        <% end %>
+
+  
+    </div>
+ 
+<section class="auth m-3 shadow-sm">
+
+<% if @entity.errors.any? %>
+  <div id="error_explanation">
+    <h2><%= pluralize(@entity.errors.count, "error") %> prohibited this model from being saved:</h2>
+    <ul>
```

Comment:
- [ ] when i try to add  a transaction I get an error kindly fix
![Screenshot from 2023-10-20 17-05-24](https://github.com/SAZZAD1Q2/BudgetMate/assets/66097371/aaf08efe-4294-49d6-8500-7802b0b8cb96)